### ISSUE TITLE
It just completely broke

### ISSUE THREAD

**Issue by pixelcmtd** [Ampless/dsbuntis#2]
While testing a new Amplessimus feature, I noticed this:
<img src="https://user-images.githubusercontent.com/35269695/110635410-57f50c00-81ab-11eb-9222-284d9dde8288.png">

**Comment by pixelcmtd** [comment]
just reproduced it in accemus:
```
$ dart run bin/accemus.dart x x
Unhandled exception:
type 'Null' is not a subtype of type 'String'
#0      getData (package:dsbuntis/src/dsbapi.dart:165:28)
<asynchronous suspension>
#1      getAllSubs (package:dsbuntis/src/dsbapi.dart:276:16)
<asynchronous suspension>
#2      main (file:///Users/chrissx/Accemus/bin/accemus.dart:9:26)
<asynchronous suspension>
```

**Comment by pixelcmtd** [comment]
i just tried reproducing this from within the tests and everything works?!

**Comment by pixelcmtd** [comment]
somehow jsonDecode returns null sometimes and sometimes not

**Comment by pixelcmtd** [comment]
ok i got a local debugging setup and now i can look at dsb's error:
```
{"Message":"Fehler bei der Verarbeitung der Anforderung.","StackTrace":"","ExceptionType":""}
```

**Comment by pixelcmtd** [comment]
seems like the old api just isnt available anymore, lets just use another one

**Comment by pixelcmtd** [comment]
fixed in 7a853a7d2a27788a27a58f28433ab244e9c30058
